### ISSUE TITLE
Test gov proposals

### ISSUE THREAD

**Issue by alpe** [confio/tgrade#291]
We need an integration test for all all supported gov proposal types.
1) Create a proposal in the the related contract
2) Vote + execute
3) Capture the response and convert into `TgradeMsg`
4) Ensure the relevant fields are set with the expected data

Goal is to ensure that the contract response object matches the expected mappings in tgrade. There were some issues in the past where the `msg` field had different name like `migrate_msg`

**Comment by BirgitPohl** [comment]
I would split this task into 4. It makes it easier to review later. A review comes with less human errors.

**Comment by alpe** [comment]
All 4 steps should be implemented into the same integration test.
first one is setup system under test, second is to run the test case, last two are verifying assumptions.
To split this into multiple tasks then you could go by proposal although they all for a contract may all go into the same table test and can not be worked on in parallel.
We have voting with OC, AP and validator community. They are more or less independent.
We have some coverage for the validator community voting already in https://github.com/confio/tgrade/blob/main/x/poe/contract/tgrade_validator_voting_integration_test.go#L27
